docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -0,0 +1,177 @@
+
+1. In the code section of the Model Builder tool, select **Added Projects** to add the projects to the solution.
+2. In solution explorer, right-click the *TaxiFarePrediction* project. Then, select **Add > Existing Item**. For file type drop down, select `All Files`, navigate to the *TaxiFarePredictionML.Model* project directory and select the `MLModel.zip` file. Then right-click the recently added `MLModel.zip` file and select *Properties*. For the Copy to Output Directory option, select *Copy if Newer* from the dropdown.
+3. Right-click *TaxiFarePrediction* project. Then, **Add > Reference**. Choose the **Projects > Solution** node and from the list, check the *TaxiFarePredictionML.Model* project and select OK.
+
+4. Open the *Program.cs* file in the *TaxiFarePrediction* project.
+5. Add the following using statements:
+
+    ```csharp
+    using System;
+    using Microsoft.ML;
+    using TaxiFarePredictionML.Model.DataModels;
+    ```
+
+6. Add the `ConsumeModel` method to the `Program` class. The `ConsumeModel` will create a `PredictionEngine` based on the model generated by Model Builder to make predictions on new data and print them out to the console.
+
+    ```csharp
+    static ModelOutput ConsumeModel(ModelInput input)
+    {
+        // 1. Load the model
+        MLContext mlContext = new MLContext();
+        ITransformer mlModel = mlContext.Model.Load("MLModel.zip", out var modelInputSchema);
+
+        // 2. Create PredictionEngine
+        var predictionEngine = mlContext.Model.CreatePredictionEngine<ModelInput, ModelOutput>(mlModel);
+
+        
+
+        // 3. Use PredictionEngine to use model on input data
+        ModelOutput result = predictionEngine.Predict(input);
+
+        // 4. Return prediction result
+        return result;
+    }
+    ```
+
+7. Add the following code to the `Main` method and run the application.
+
+    ```csharp
+    // Create sample data
+    ModelInput input = new ModelInput()
+    {
+        Vendor_id = "CMT",
+        Rate_code = 1,
+        Passenger_count = 1,
+        Trip_time_in_secs = 1271,
+        Trip_distance = 3.8f,
+        Payment_type = "CRD"
+    };
+
+    // Make prediction
+    ModelOutput prediction = ConsumeModel(input);
+
+    // Print Prediction
+    Console.WriteLine($"Predicted Fare: {prediction.Score}");
+    Console.ReadKey();
+    ```
+
+    The output generated by the program should look similar to the snippet below:
+
+    ```bash
+    Predicted Fare: 16.82245
+    ```
+
+## Next Steps
+
+In this tutorial, you learned how to:
+> [!div class="checklist"]
+> * Prepare and understand the data
+> * Choose a scenario
+> * Load the data
+> * Train the model
+> * Evaluate the model
+> * Use the model for predictions
+
+- [Deploy a model to Azure Functions](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net)
```

Comment:
Should this be here?

Reply:
Maybe. In those how-to's we just require that the user has a pre-trained model which is what they'll be building with this tutorial.

Reply:
Suggest making it consistent with the other tutorials i.e. In this tutorial, you *learned* how to
- blah

Then have a sentence about what to do next
- with a button linking to the next thing

See attached screenshotScreen Shot 2019-06-10 at 4.57.17 PM

Reply:
#resolved